data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -206,13 +206,19 @@
 EJSCREEN_AREAS_OF_CONCERN_STATE_95TH_PERCENTILE_COMMUNITIES_FIELD = (
     "EJSCREEN Areas of Concern, State, 95th percentile (communities)"
 )
-
 # Mapping inequality data.
 HOLC_GRADE_D_TRACT_PERCENT_FIELD: str = "Percent of tract that is HOLC Grade D"
 HOLC_GRADE_D_TRACT_20_PERCENT_FIELD: str = "Tract is >20% HOLC Grade D"
 HOLC_GRADE_D_TRACT_50_PERCENT_FIELD: str = "Tract is >50% HOLC Grade D"
 HOLC_GRADE_D_TRACT_75_PERCENT_FIELD: str = "Tract is >75% HOLC Grade D"
 
+# Michigan Environmental Screening Tool ETL Constants
+MICHIGAN_EJSCREEN_SCORE_FIELD: str = "Michigan EJSCREEN Score Field"
+MICHIGAN_EJSCREEN_PERCENTILE_FIELD: str = "Michigan EJSCREEN Percentile Field"
+MICHIGAN_EJSCREEN_PRIORITY_COMMUNITY_FIELD: str = (
+    "Michigan EJSCREEN Priority Community"
+)
+MICHIGAN_EJSCREEN_PRIORITY_COMMUNITY_THRESHOLD: float = 0.75
```

Comment:
Nit: I believe the other constants in this file are all exclusively used for naming things, not for logic like this. (Even the PERCENTILE field in here is only used for naming things.) 

Would you think it's a good idea to move this back inside the class? thanks!

Reply:
Yes, upon review, it looks like the other fieldnames are references to string types. I have moved it back as an instance variable in the class.